include/openthread/srp_server.h

```diff
@@ -113,7 +113,23 @@ otError otSrpServerSetDomain(otInstance *aInstance, const char *aDomain);
 void otSrpServerSetEnabled(otInstance *aInstance, bool aEnabled);
 
 /**
- * This method sets LEASE & KEY-LEASE range that is acceptable by the SRP server.
+ * This function returns LEASE & KEY-LEASE range from the SRP server.
+ *
+ * @param[in]   aInstance     A pointer to an OpenThread instance.
+ * @param[out]  aMinLease     A pointer to the minimum LEASE interval in seconds.
+ * @param[out]  aMaxLease     A pointer to the maximum LEASE interval in seconds.
+ * @param[out]  aMinKeyLease  A pointer to the minimum KEY-LEASE interval in seconds.
+ * @param[out]  aMaxKeyLease  A pointer to the maximum KEY-LEASE interval in seconds.
+ *
+ */
+void otSrpServerGetLeaseRange(otInstance *aInstance,
+                              uint32_t *  aMinLease,
+                              uint32_t *  aMaxLease,
+                              uint32_t *  aMinKeyLease,
+                              uint32_t *  aMaxKeyLease);
```

Comment:
Rather than having a parameter for each value, what about defining a struct that is passed as a single param?

Reply:
@jwhui Udpated to use a `otSrpServerLeaseConfig` struct. Also defined an internal `Srp::Server::LeaseConfig` class with convenient utility methods (e..g `IsValid`, `GrantLease` and `GrantKeyLease`).